### PR TITLE
Desktop view [Portfolio project]

### PR DESCRIPTION
**Pull request by mwafrika**
### Mobile first responsive design

desktop2

### CODE REVIEW

File: src/home/index.html [hunk 1]
```diff
@@ -8,268 +8,308 @@
...
+            </ul>
+          </div>
+        </section>
+      </header>
+      <section class="works">
+        <div class="card-1">
+          <div class="card-1-2">
+            <div class="image-container"></div>
+          </div>
+          <div class="card-1-1">
```
Comment (MrBrN197) [inline src/home/index.html:85]: - [ ] Please ensure that your styling in the desktop version matches the Figma design in the card section.

|Figma Design|Your Design|
|---|---|
|image|image|

File: src/home/index.html [hunk 2]
```diff
@@ -8,268 +8,308 @@
...
+            <li class="back">
+              <a href="#"><img src="../assert/images/html.png" alt="HTML" /></a>
+              HTML
             </li>
-            <li class="lang">
-              <img src="../assert/images/html.png" alt="javascript" />
-              <p>HTML</p>
+            <li class="back">
+              <a href="#"><img src="../assert/images/css.png" alt="CSS" /></a>
+              CSS
```
Comment (MrBrN197) [inline src/home/index.html:245]: - [ ] Please also try to match the Figma design here as closely as possible. notice that the html, CSS, javascript badges have a lighter background.
- [ ]  These badges also have equal widths, notice that your "HTML" and "CSS" badges have different widths than the javascript badge.

|Figma Design|Figma Design|
|---|---|
|image|image|

File: src/home/index.html [hunk 3]
```diff
@@ -8,268 +8,308 @@
...
+          </div>
         </div>
-        <div class="langage-list">
+
+        <div class="about-img">
+          <!-- <img src="../assert/images/try.svg" alt="About" /> -->
+          <div class="lang title">
+            <h2>Languages</h2>
+            <img src="../assert/images/chevron-v.png" alt="Language" />
+          </div>
```
Comment (MrBrN197) [inline src/home/index.html:230]: - [ ] Hello please notice and try to use the correct `font-color` and `font-weight` for the text "Languages"

|Figma Design|Your Design|
|---|---|
|image|image|

- [ ] Please make the same change for the other text in this section (that is the "Frameworks" and "Skills" text).

|Figma Design|Your Design|
|---|---|
|image|image|

**Review by MrBrN197 — CHANGES_REQUESTED** [review]
## Hi,  @mwafrika, @L2oukmane  & @Kayonga99  👋 

You've done a really nice job so far.
There are some issues that you still need to work on to go to the next project but you are almost there!
## To highlight 👏👏
- ✔ Great animations on the page 👏

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/home/index.html (hunk 3 above)
Comment (mwafrika) [reply src/home/index.html:230]: Yeah. I've done already, you can have a look sir

File: src/home/index.html (hunk 2 above)
Comment (mwafrika) [reply src/home/index.html:245]: Done!

File: src/home/index.html (hunk 1 above)
Comment (mwafrika) [reply src/home/index.html:85]: Done already!

**Comment by mwafrika** [comment]
I have updated the screenshot. Kindly check it again! thanks

**Pull request merged**